packages/table/src/regions.ts

```diff
@@ -32,6 +37,11 @@ export enum RegionCardinality {
      * A region that represents all cells in the table.
      */
     FULL_TABLE,
+
+    /**
+     * A region that represents all row header cells.
+     */
+    ROW_HEADERS,
```

Comment:
:stop_sign: are these new values supported everywhere `RegionCardinality` is used?